package.json

```diff
@@ -98,6 +98,9 @@
     "typechain": "^5.0.0",
     "typescript": "^4.2.2"
   },
+  "overrides": {
```

Comment:
@lynnshaoyu  can we not just add `sdk-core` to our real dependency array and use the latest version?

Reply:
https://github.com/Uniswap/smart-order-router/pull/261#discussion_r1235929311

Reply:
i'm still not sure this is right, since `smart-order-router` does directly depend on `sdk-core` i think it should be listed as a dependency

Reply:
I just fixed this